lib/index.d.ts

```diff
@@ -611,11 +648,13 @@ export interface Settings {
   angularSleepToleranceSqr: number;
 }
 
-export let internal: {
-  Settings: Settings;
-  Sweep: Sweep;
-  Manifold: Manifold;
-  Distance: any;
-  TimeOfImpact: any;
-  DynamicTree: DynamicTree;
-};
+export namespace internal {
+  let Settings: Settings;
```

Comment:
Don't you also have to also export this and the other objects?

Reply:
I didn't need them for my use case, so I did not add their constructors. I can see if I find the time to have a look into that as well, but some types are not typed properly, such as `Distance` and `TimeOfImpact`, so adding them is extra work (and possibly extra bugs).

Reply:
I just meant to add the `export` keyword in front, at least for `setting`, `Distance` and `TimeOfImpact` because I think these typings should be right (well `any` is always right). You don't have to bother with fixing the other types of `internal`, I know the typings are wrong and I will fix these in another PR :)

Reply:
done